.gitignore

```diff
@@ -0,0 +1,14 @@
+/venv
+
+**/__pycache__/
+**/.pytest_cache
+**/*.pyc
+
+# ignore local pyenv settings
+.python-version
```

Comment:
This should probably be put in a global (private) .gitignore as it is not something that is required for the repo.